aries-site/src/layouts/content/DocsPageHeader.js

```diff
@@ -27,11 +27,13 @@ export const DocsPageHeader = ({ title, topic, render }) => {
           </Link>
         )
       }
-      subtitle={<SubsectionText>{page.description}</SubsectionText>}
+      subtitle={
+        <SubsectionText accessibility={page.accessibility}>
+          {page.description}
+        </SubsectionText>
+      }
       margin={{ bottom: 'small' }}
-    >
-      {page.status && <Status status={page.status} />}
```

Comment:
Why are we getting rid of status here?

Reply:
this was just old code I can put back it was being unused it was from when we started the site we had statuses on is complete in grommet/figma we dont use this anymore

Reply:
Oh gotcha, if we don't use it anymore seems fine to get rid of

Reply:
@taysea your thoughts. I can keep for us for now or remove I know its been a while.